lte/cloud/go/services/subscriberdb/storage/digest.go

```diff
@@ -166,18 +167,16 @@ func (l *digestLookup) DeleteDigests(networks []string) error {
 }
 
 // GetDigest returns the digest information for a particular network.
-func GetDigest(l DigestLookup, network string) (string, int64, error) {
+func GetDigest(l DigestLookup, network string) (string, error) {
 	digestInfos, err := l.GetDigests([]string{network}, clock.Now().Unix())
 	if err != nil {
-		return "", 0, err
+		return "", err
 	}
-	// There should be at most 1 digest for each network
-	// if digest not found, return default value
+	// Each network has at most 1 digest; if digest not set yet, return empty digest
```

Comment:
I like "...; if digest not found, ..." since it more accurately describes the actual experience